core/profiles/standard/standard.install

```diff
@@ -440,17 +434,25 @@ function standard_install() {
   // Update the menu router information.
   menu_rebuild();
 
-  // Enable the admin theme.
-  db_update('system')
-    ->fields(array('status' => 1))
-    ->condition('type', 'theme')
-    ->condition('name', 'seven')
-    ->execute();
-  config_set('system.core', 'admin_theme', 'seven');
-  config_set('system.core', 'node_admin_theme', '1');
+  $default_theme = 'basis';
+  $admin_theme = 'seven';
+  // Enable the Basis and Seven themes, and disable the Stark theme.
+  theme_enable(array($default_theme, $admin_theme));
+  theme_disable(array('stark'));
+  config('system.core')
```

Comment:
same thing here...  especially since we have comments interspersed.
```php
$config = config('system.core');
// Set the Basis theme as the default theme.
// @see system_install()
$config->set('theme_default', $default_theme);
// Set the Seven theme as the admin theme.
$config->set('admin_theme', $admin_theme);
$config->set('admin_theme_same_as_default', 0);
$config->set('node_admin_theme', 1);
```

See my note about the save, below :)

Reply:
...oooh, there's more 😅